packages/esm-patient-orders-app/src/components/orders-details-table.component.tsx

```diff
@@ -156,7 +161,7 @@ const OrderDetailsTable: React.FC<OrderDetailsProps> = ({ title, patientUuid, sh
           items={orders}
           setOrderItems={setOrders}
           openOrderBasket={launchOrderBasket}
-          openOrderForm={launchOrderBasket}
+          openOrderForm={order.type === 'drugorder' ? launchAddDrugOrder : launchAddLabsOrder}
```

Comment:
We are currently working on radiology and procedure orders and I guess these will also be visualized through this component. We can refine the ternary statement to not default to lab form

Reply:
That sounds like a nice enhancement. Will update this PR to add the provision for this